documents/src/pages/elements/slider.md

```diff
@@ -141,8 +147,224 @@ ef-slider{
 <ef-slider min="0" max="100" from="60" to="80" step="20" range show-steps show-input-field></ef-slider>
 ```
 
+## Markers
+You can show markers to provide more context to users on any specific values of Slider. The markers can show with or without label.
+
+Defines each marker with `ef-slider-marker`. Position of the marker is set by `value`.
+
+::
+```javascript
+::slider::
+```
+```css
+ef-slider {
+  width: 50%;
+  margin: 10px;
+}
+
+#tracking-speed {
+  --progress-color: transparent;
+}
+
+.container {
+  margin: 20px;
+  margin-left: 30px;
+  margin-bottom: 35px;
+}
+
+```
+```html
+<div class="container">
+  <label for="temperature">Temperature</label>
+  <ef-slider id="temperature" value="70">
+    <ef-slider-marker value="0">0°C</ef-slider-marker>
+    <ef-slider-marker value="20">20°C</ef-slider-marker>
+    <ef-slider-marker value="70">70°C</ef-slider-marker>
+    <ef-slider-marker value="100">100°C</ef-slider-marker>
+  </ef-slider>
+</div>
+<div class="container">
+  <label for="tracking-speed">Tracking Speed</label>
+  <ef-slider id="tracking-speed" step="25" value="50">
+    <ef-slider-marker value="0">Slow</ef-slider-marker>
+    <ef-slider-marker value="25"></ef-slider-marker>
+    <ef-slider-marker value="50">Medium</ef-slider-marker>
+    <ef-slider-marker value="75"></ef-slider-marker>
+    <ef-slider-marker value="100">Fast</ef-slider-marker>
+  </ef-slider>  
+</div>
+```
+::
+
+```html
+<ef-slider id="temperature" value="70">
+  <ef-slider-marker value="0">0°C</ef-slider-marker>
+  <ef-slider-marker value="20">20°C</ef-slider-marker>
+  <ef-slider-marker value="70">70°C</ef-slider-marker>
+  <ef-slider-marker value="100">100°C</ef-slider-marker>
+</ef-slider>
+```
+
+Markers can be used with `step` for a similar use case as radio button but in Slider style. Typically, with stepped Slider, you would need to hide the progress when users drag the Slider.
```

Comment:
I'm not sure why we need to mention if slider is using `step` or not to hide the slider, step or not progress can be hidden using following code. Proposing a simpler version. 
```suggestion
Markers can be used with `step` for a similar use case as radio button but in Slider style. To hide slider progress, set `--progress-color` CSS variables to `transparent`.
```